tests/dogtag/pytest-ansible/pytest/performance_test/test_cert_revocation.py

```diff
@@ -36,64 +36,74 @@
 
 args = parser.parse_args()
 
-# Create a PKIConnection object that stores the details of the CA.
-connection = PKIConnection('https', args.hostname, args.port, cert_paths=args.ca_cert_path)
 
-# The pem file used for authentication. Created from a p12 file using the
-# command -
-# openssl pkcs12 -in <p12_file_path> -out /tmp/auth.pem -nodes
-connection.set_authentication_cert(args.client_cert)
-
-# Instantiate the CertClient
-cert_client = CertClient(connection)
-
-
-def run_test(cert_sn, number_of_tests_per_thread):
-    # execute the specified number of tests
-    for sn in range(number_of_tests_per_thread):
-        try:
-            start = timer()
-            log.info("Revoking Cert : {}".format(cert_sn[sn]))
-            revoke_data = cert_client.revoke_cert(cert_sn[sn], revocation_reason='Key_Compromise')
-            end = timer()
-            revocation_times.append(int(end - start))
-        except Exception as error:
-            log.error(error)
+class TestClient(threading.Thread):
+
+    def __init__(self, connection, cert_sn, number_of_tests_per_client):
+        super().__init__()
+        self.connection = connection
+        self.number_of_tests_per_client = number_of_tests_per_client
+        self.cert_sn = cert_sn
+        self.cert_client = CertClient(self.connection)
+        return
+
+    def run(self):
+        # execute the specified number of tests
+        for sn in range(self.number_of_tests_per_client):
+            try:
+                start = timer()
+                log.info("Revoking Cert : {}".format(self.cert_sn[sn]))
+                revoke_data = self.cert_client.revoke_cert(self.cert_sn[sn], revocation_reason='Key_Compromise')
+                end = timer()
+                revocation_times.append(int(end - start))
```

Comment:
Not sure if `int()` is required, but the enrollment test does not use it. I think they should be consistent.